<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureEditorToolBar</class>
 <widget class="QWidget" name="TextureEditorToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>29</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="layoutZoom">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="labelZoom">
       <property name="text">
        <string>Zoom </string>
       </property>
       <property name="buddy">
        <cstring>zoomToFit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="zoomToFit">
       <property name="toolTip">
        <string>Fit Window</string>
       </property>
       <property name="icon">
        <iconset theme="zoom-fit-best">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="zoom">
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>27</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::PlusMinus</enum>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5000</number>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="flipVertically">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>Flip Vert.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="filter">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelLevel">
     <property name="text">
      <string>Level</string>
     </property>
     <property name="buddy">
      <cstring>level</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="level">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>27</height>
      </size>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelLayer">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>  Layer</string>
     </property>
     <property name="buddy">
      <cstring>layer</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="layer">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>27</height>
      </size>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelSample">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>  Sample</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="sample">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>27</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelFace">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>27</height>
      </size>
     </property>
     <property name="text">
      <string>  Face</string>
     </property>
     <property name="buddy">
      <cstring>face</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="face">
     <item>
      <property name="text">
       <string>Positive X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Negative X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Positive Y</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Negative Y</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Positive Z</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Negative Z</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>8</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
